### ISSUE TITLE
Disallow same label workspace at database level

### ISSUE THREAD

**Issue by inkhey** [tracim/tracim#1096]
**Describe the bug**
According to fix for #1093 , workspace could not anymore be created with same label. this fix is a high level verification, we need to check this at lower level too to avoid incoherent database.

- [ ] Make label column unique in Workspace sqlalchemy 
- [ ] find solution for possible migration issue ?

**Comment by lebouquetin** [comment]
There is a problem to decide:

- for example two users may want to have a dedicated workspace `Perso`
- if we authorize this, what about webdav in case two workspaces have the same label?

**Comment by lebouquetin** [comment]
This issue requires a discussion before any work

**Comment by inkhey** [comment]
another point to thing about : 
- Behavior between mysql and postgresql/sqlite seems different, mysql is not case sensitive whereas sqlite/postgresql are.

**Comment by inkhey** [comment]
deprecated dues to new issue #2477
